NEWS.md

```diff
@@ -1,3 +1,10 @@
+# travis 0.2.11.9002
```

Comment:
NEWS are generated by {fledge} here. This requires a specific format for top-level commits.

Reply:
I tried {fledge} for some repos. I like it but find it problematic for repos with many collabs since everyone has to agree on the format OR there can only be one person merging PR.
Otherwise the concept won't work.

I found myself switching between {flege} and "the usual way" and confused myself more than it helped me :/ 
Also I usually follow the "squash merge" principle rather than merging all single commits into master of a PR.

Reply:
Thanks for the feedback. Everyone has to format merge commits in a special way, yes. If they don't, the NEWS still can be edited manually. Maybe {fledge} should give a summary of merge commits that don't have a corresponding message?

Squashing commits generates extra work on the contributor's end when they have branches that are built on top each other. What's a good workflow to handle this?

Reply:
> Squashing commits generates extra work on the contributor's end when they have branches that are built on top each other. What's a good workflow to handle this?

I usually don't do this but can think of this causing issues.
I don't know what the best WF is, probably there is no "best" and all have their pros and cons.
With {mlr} and friends I got used to squashing because we only do this with PRs. 

Happy to adjust for {travis} and {tic}. However, for me this always comes with the overhead to think about this again when working on these repos. 
I might simply need more practice on this.

Another thing is that I usually do not give too much thought BEFORE the commit message if this commit will become a top-level commit. My WF usually goes with the "PR-approach" meaning that all PRs are top-level commits with newsworthy changes.

How can we find the best middle way here? Do not want to work against your WF :)